packages/edit-widgets/src/components/widget-areas-block-editor-content/index.js

```diff
@@ -21,20 +21,22 @@ export default function WidgetAreasBlockEditorContent( {
 	blockEditorSettings,
 } ) {
 	return (
-		<BlockTools>
+		<>
 			<KeyboardShortcuts />
 			<BlockEditorKeyboardShortcuts />
-			<Notices />
 			<div className="edit-widgets-block-editor editor-styles-wrapper">
-				<EditorStyles styles={ blockEditorSettings.styles } />
-				<BlockSelectionClearer>
-					<WritingFlow>
-						<ObserveTyping>
-							<BlockList className="edit-widgets-main-block-list" />
-						</ObserveTyping>
-					</WritingFlow>
-				</BlockSelectionClearer>
+				<BlockTools>
```

Comment:
BlockTools should be wrapped around the editor canvas (`.editor-styles-wrapper`). It should not be part of the canvas.